addons/html_builder/static/src/builder/plugins/grid_layout/grid_layout_plugin.js

```diff
@@ -0,0 +1,68 @@
+import { Plugin } from "@html_editor/plugin";
+import { setElementToMaxZindex } from "@html_builder/builder/utils/grid_layout_utils";
+import { isMobileView } from "@html_builder/builder/utils/utils";
+
+const gridItemSelector = ".row.o_grid_mode > div.o_grid_item";
+
+export class GridLayoutPlugin extends Plugin {
+    static id = "gridLayout";
+    static dependencies = ["overlay", "history"];
+    resources = {
+        has_overlay_buttons: this.isGridItem.bind(this),
+        get_overlay_buttons: this.getActiveOverlayButtons.bind(this),
+    };
+
+    setup() {
+        this.target = null;
+    }
+
+    getActiveOverlayButtons({ target, callback }) {
+        const buttons = [];
+        if (this.isGridItem(target)) {
+            this.target = target;
+            const isMobile = isMobileView(this.target);
+            if (!isMobile) {
```

Comment:
isDesktop ? or something else ?

Reply:
or if isMobile() { return []}

Reply:
For this one, I prefer how it is now it is easier to follow the flow :slightly_smiling_face: